manifest.tmpl

```diff
@@ -0,0 +1,12 @@
+image: rancher/hardened-build-base:{{build.tag}}
+manifests:
+  -
+    image: rancher/hardened-build-base:{{build.tag}}-amd64
+    platform:
+      architecture: amd64
+      os: linux
+  -
+    image: rancher/hardened-build-base:{{build.tag}}-s390x
```

Comment:
The s390 image is not "hardened" therefore we should not induce confusion by naming it such.